recipes/wayland-protocols/all/conandata.yml

```diff
@@ -1,22 +1,22 @@
 sources:
   "1.31":
-    url: "https://github.com/wayland-project/wayland-protocols/archive/1.31.tar.gz"
-    sha256: "04d3f66eca99d638ec8dbfdfdf79334290e22028f7d0b04c7034d9ef18a3248a"
+    url: "https://gitlab.freedesktop.org/wayland/wayland-protocols/-/archive/1.31/wayland-protocols-1.31.tar.bz2"
+    sha256: "3d7fa65a6f3c86747f780a628df04d3c7a981776dee2b6a492cc112b3bbe2820"
   "1.27":
-    url: "https://github.com/wayland-project/wayland-protocols/archive/1.27.tar.gz"
-    sha256: "6dd6ee86478adf4347f3b8b4f3da62dbe9e44912c9cef21cf99abfe692313df4"
-  "1.26":
-    url: "https://github.com/wayland-project/wayland-protocols/archive/1.26.tar.gz"
-    sha256: "fe56386f436a84e97c3b6a61b76306f205a64425900f247ad0048174b9c32d4d"
+    url: "https://gitlab.freedesktop.org/wayland/wayland-protocols/-/archive/1.27/wayland-protocols-1.27.tar.bz2"
```

Comment:
Prefer official tarball for these 2 ones. It's always better than tarball generated on the fly by some service.